pywinauto/keyboard.py

```diff
@@ -605,22 +606,29 @@ def parse_keys(string,
             while modifiers:
                 if DEBUG:
                     print("MODS-", modifiers)
-                keys.append(VirtualKeyAction(modifiers.pop(), down = False))
+                keys.append(VirtualKeyAction(modifiers.pop(), down=False))
 
         # just in case there were any modifiers left pressed - release them
         while modifiers:
-            keys.append(VirtualKeyAction(modifiers.pop(), down = False))
+            keys.append(VirtualKeyAction(modifiers.pop(), down=False))
 
         return keys
 
+
     def LoByte(val):
         """Return the low byte of the value"""
         return val & 0xff
 
+
     def HiByte(val):
         """Return the high byte of the value"""
         return (val & 0xff00) >> 8
 
+    def _run(keys, pause):
+        for k in keys:
+            k.run()
+            time.sleep(pause)
+
     def SendKeys(keys,

```

Comment:
Since you've started improving this module, can I ask you to rename `SendKeys` to `send_keys`? And make alias `SendKeys = deprecated(send_keys)` where `deprecated` should be imported so: `from . import deprecated`.

Reply:
Will do.